storage/pkg/flagset/driverowncloud.go

```diff
@@ -2,59 +2,59 @@ package flagset
 
 import (
 	"github.com/micro/cli/v2"
-	"github.com/owncloud/ocis/ocis-reva/pkg/config"
+	"github.com/owncloud/ocis/storage/pkg/config"
 )
 
 // DriverOwnCloudWithConfig applies cfg to the root flagset
 func DriverOwnCloudWithConfig(cfg *config.Config) []cli.Flag {
 	return []cli.Flag{
 		&cli.StringFlag{
 			Name:        "storage-owncloud-datadir",
-			Value:       "/var/tmp/reva/data",
+			Value:       "/var/tmp/ocis/data",
```

Comment:
`/var/tmp/ocis/owncloud` similar to `local`

Reply:
fine, thx